packages/core/src/components/account/orders/MyAccountListOrders/MyAccountListOrdersEmptyState/MyAccountListOrdersEmptyState.tsx

```diff
@@ -0,0 +1,19 @@
+import Link from 'next/link'
+import { Icon } from '@faststore/ui'
+
+import styles from './styles.module.scss'
+
+export default function MyAccountListOrdersEmptyState() {
+  return (
+    <div className={styles.container}>
+      <div className={styles.iconContainer}>
+        <Icon name="Bag2" width={56} height={56} />
+        <h2 className={styles.title}>You don’t have any orders</h2>
+      </div>
+
+      <Link href="/" className={styles.link}>
+        Start shopping
+      </Link>
+    </div>
```

Comment:
Hmm, we are using data-attributes to style to keep the FastStore pattern instead of using CSS modules (although I don't like data-attributes hahah). Can you follow with data-attr? The other style files have the pattern, but you can find more details here https://developers.vtex.com/docs/guides/faststore/using-themes-components